README.md

```diff
@@ -25,63 +36,53 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
+   - Arduinos aren't perfect I2C target devices. Particularly, the Arduino's Wire library doesn't support [clock stretching](https://onlinedocs.microchip.com/pr/GUID-CA6B1F8F-E09D-4780-A52A-CBBF61902464-en-US-2/GUID-5CCAB0DB-28BD-4095-B2E2-2F3CF0FC6966.html) in a way that allows the target to finish reacting to the previous command if it hasn't done so yet before the transmission occurred. That's why it's important to make sure that the **target is not flooded** with commands or requests with too little time to handle them. I2Cwrapper provides an [adjustable minimum delay](#adjusting-the-i2c-delay) between transmissions to handle that problem.
 
 ### Limitations for module authors
 
    - No **serialization protocol** is used at the moment, so the implementation is machine dependent in regard to the endians and sizes of data types. Modules will have to take care that transmitted commands and requests will transmit defined amounts of bytes by using typeguards for ambiguously sized datatypes like *int*.
    - Modules use one byte **command codes** for each distinct function call to the target. At the moment, no mechanism is in place to prevent newly developed modules from reusing codes already used by another module or by one of the I2Cwrapper core functions. Note that this will only lead to problems if two modules are used concurrently by a target device. The [I2Cwrapper documentation](https://ftjuh.github.io/I2Cwrapper/docs/class_i2_cwrapper.html) has a list of code ranges used by the currently available modules. Strictly reserved ranges are 0-9 and 240-255.
 
-See [below](#How to add new modules) if you are interested in writing a new module and implementing your own target device.
+See [below](#how-to-add-new-modules) if you are interested in writing a new module and implementing your own target device.
 
 # Usage
 
 ## Installation
 
 ~~Install I2Cwrapper from the Arduino library manager (not available yet).~~ Until I2Cwrapper is picked up by the Arduino library manager, you'll have to [install it manually](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#manual-installation) by copying the I2Cwrapper folder to your Arduino library folder and (re)starting the Arduino software.  You'll find an I2Cwrapper examples folder after successful installation.
 
-## Configuring and uploading the firmware
+## Target side: configuring and uploading the firmware
```

Comment:
If we use "Target side: " here, we'll need to add "Controller side: " somewhere below, so I think "Configuring and uploading the firmware to the target device" has fewer side effects.

Reply:
reverting.